### DISCUSSION TITLE
Controversial change of limiting per round missions to 3 max. What if with factions update, your reputation affected that value?

### DISCUSSION THREAD

**Discussion by IgRAzm** [FakeFishGames/Barotrauma#11112] [Ideas and suggestions]
i'm talking as a player with no experience in features from unstable. but i've read complaints to the 3 limit for campaign missions, and thought - the core argument of the players in favour of the change is that the earlygame pacing is accelerated too much with more missions. so, thinking of anything that doesn't start out at very high value to try and connect to the mission count, i've thought of crew reputation. i can agree that it is snowballed very much by the higher mission setting from pre patch as each hard mission additively increases it but relatively safe ones also do. in truth, reputation isnt used quite as much as it could be, in stable version of game atl, so it feels like 2-3 would be a reasonable mission starting point to have a gradual income growth in the earlygame when you are unknown and making ends meet so to speak, but now with the potential to increase it.

i think individual outpost reputation would slightly affect the mission count so that when you're revered you get +1 for the round when you leave. but after 1.0 i believe there's gonna be not just coalition but separatist and other faction outposts. you need to earn favor throughout campaign, and only at the highest rep, you may get extra missions, but only in the faction outposts that recognise your authority.

so basically like quests in rpgs - you dynamically unlock more job opportunities. this would be a very good incentive for crews to keep an appealing image; if you rob every station you come across, only the most desparate will support or tolerate you in any way and that includes job offerings.

**Comment by Limblicious** [comment]
Typically reputation grinds unlock rewards, not opportunities to grind more rewards. Look at any RPG and see that when you successfully hit a reputation milestone you unlock an ability, a class, a weapon, a new area, or new quests. 

Unlocking the ability to simply do more of the same missions doesn't make sense. 

The reason that this change is being criticized is because it was **_optional_** to increase your max mission allowance. It wasn't required. The default setting is 2. Don't like more missions? Don't take 'em. But for the people who enjoyed the added chaos and difficulty of taking a lot of missions are seeing that option taken away with no reason and no replacement. That's just poor design/decision making.

The people for the change are either so buried in mods they aren't playing the same game anymore (I don't see how you could get 10 missions from a new game start first outpost without mods and the resources to DO all of them) or do not understand the word "option".

Should ask the devs what their favorite mission/sub/class/weapon is and just delete all the other ones if we're on the limiting gameplay loops boat lol.
